### PR TITLE
Update readme.md RC1

### PR DESCRIPTION
**Pull request by workgroupengineering**
## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ![Star our repo to show support](https://user-images.githubusercontent.com/552074/235945895-1b896994-a0b6-4e7c-a522-c5688c4ec1b9.png)
-![Header](https://user-images.githubusercontent.com/552074/235865745-2a8e7274-4f66-4f77-8f05-feeb76e7d478.png)
+![Header](https://github.com/AvaloniaUI/Avalonia/assets/12531229/daef336b-90e6-4620-8c80-abbcd28d5599)
+
+**[📖 Dettails](https://github.com/AvaloniaUI/Avalonia/discussions/11593)**
```
Comment (IhateTrains) [inline readme.md:4]:
```suggestion
**[📖 Details](https://github.com/AvaloniaUI/Avalonia/discussions/11593)** 
```

**Review by IhateTrains — APPROVED** [review]

**Comment by MikeCodesDotNET** [comment]
These links are useful, but I think they would be better in the Learning section, given the current flow of the readme.

**Comment by robloo** [comment]
My Two Cents:

The Porting Guide seems like a good idea to include. However, I don't think a discussion "details" link like should included. If anything a new section should be added for "Current Version" that includes:

 1. A link to the https://avaloniaui.net/Blog -- and perhaps the specific RC announcement if absolutely necessary
 2. The links to the porting guides (there are two so far)
     - https://github.com/AvaloniaUI/Avalonia/wiki/Avalonia-11-Porting-Guide
     - https://github.com/AvaloniaUI/Avalonia/wiki/Porting-to-Control-Themes

```md
## Version 11

For latest announcements see the [Avalonia blog](https://avaloniaui.net/Blog).

 * Converting from previous versions
   * **[📖 Porting Guide](https://github.com/AvaloniaUI/Avalonia/wiki/Avalonia-11-Porting-Guide)**
   * **[📖 Porting to Control Themes](https://github.com/AvaloniaUI/Avalonia/wiki/Porting-to-Control-Themes)**

```

A learning section would be fine too. Just throwing these links at the top seems out of place.

**Comment by workgroupengineering** [comment]
> These links are useful, but I think they would be better in the Learning section, given the current flow of the readme.
> 
> I'm also not keen to replace the header image. While we're excited for v11, I don't think we need to tweak the existing hero section.

ok. close the PR

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0036181-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Pull request closed without merging**